src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCFUnitTest.java

```diff
@@ -0,0 +1,96 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
+
+import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ReblockGVCFUnitTest {
```

Comment:
Thanks for adding these tests.  Complicated functions like these have a tendency to lose bits of logic later on if there aren't direct tests for them.